### ISSUE TITLE
Use curl -v when 404 is expected

### ISSUE THREAD

**Issue by rafabene** [redhat-developer-demos/knative-tutorial#12]
At https://redhat-developer-demos.github.io/knative-tutorial/knative-tutorial/dev/02configs-and-routes.html#crtd-invoke-service Is says that 404 is expected, but no output is show.
- Change to `curl -v` to show the 404 error.
- Highlight that 404 is expected. The sentence "The command will return HTTP 404 as there are no routes deployed yet" should come before the invocation of the curl command as I was reading the document and then I didn't see this sentence. I thought that 404 was an error and I didn't proceed the reading until solve the "problem".

**Comment by rafabene** [comment]
The explanation that an error will occur should come first. This happened again with the sentence: " will throw an error as there are no knative services deployed as part of the configuration + routes based deployment strategy." But I had already invoked and I thought I did something wrong. So I read the documents backwards instead of forward.

**Comment by kameshsampath** [comment]
closed vi #13 

@rafabene  - please validate; I will close the same once I hear from you

**Comment by rafabene** [comment]
Ok. I might be able to try until the end of the next week. Thanks @kameshsampath

**Comment by rafabene** [comment]
You can merge together with #11 so I can try it "live". It's hard to follow a large diff.
